pkg/commands/compute/serve.go

```diff
@@ -456,40 +462,27 @@ func installViceroy(
 		// Viceroy is already installed, so we check if the installed version matches the latest.
 		// But we'll skip that check if the TTL for the Viceroy LastChecked hasn't expired.
 
-		stale := g.Config.Viceroy.LastChecked != "" && g.Config.Viceroy.LatestVersion != "" && check.Stale(g.Config.Viceroy.LastChecked, g.Config.Viceroy.TTL)
+		stale := check.Stale(g.Config.Viceroy.LastChecked, g.Config.Viceroy.TTL)
```

Comment:
Just confirming that the latest version no longer matters here?

Reply:
Oh, is it because it is being checked below?

Reply:
So this was me trying to address a bug I had noticed where my config (for whatever reason, I don't know why or when it happened) had ended up with empty string values for both `LastChecked` and `LatestVersion` and because of the conditional check we had, which was...

```go
g.Config.Viceroy.LastChecked != "" && g.Config.Viceroy.LatestVersion != "" && check.Stale(g.Config.Viceroy.LastChecked, g.Config.Viceroy.TTL)
```

It meant I was never checking if the TTL was stale (and so I was left with a message telling me the TTL hadn't expired when I knew it had).

Now, in this PR, if we just pass the empty value to `check.Stale()` then it's going to calculate `LastChecked` as `time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC)` and the TTL will definitely have passed by that and so it will force a recheck for a newer Viceroy version.